third-party/realdds/include/realdds/dds-stream.h

```diff
@@ -16,11 +16,14 @@ namespace topics {
 namespace device {
     class image;
 } // namespace device
+class flexible_msg;
 } // namespace topics
 
 class dds_subscriber;
 class dds_topic_reader;
 
+class frame_metadata_syncer;

```

Comment:
in realdds?

Reply:
Removing. Implementation moved to LibRS level